src/components/Calculator.js

```diff
@@ -5,32 +5,26 @@ import CalcBtnsPad from './CalcBtnsPad';
 import CalcBtn from './CalcBtn';
 import CalcProvider from '../context/CalcContext';
 /* eslint-disable react/no-array-index-key */
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code. Please enable **all** ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.